src/components/HotelCard.jsx

```diff
@@ -14,6 +15,19 @@ function HotelCard({
   availabilityFrom,
   availabilityTo,
 }) {
+  function formatUnixToNaturalDate(dateUnix) {
+    let dateNatural = new Date(dateUnix);
+    return `${dateNatural.getDate()}/${dateNatural.getMonth()}/${dateNatural.getFullYear()}`;
```

Comment:
añadir +1 a `getMonth()` por que devuelve el indice del mes, empezando desde 0. Enero es 0, Febrero es 1 and so on. Remember [to check the docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getMonth)! 
```suggestion
    return `${dateNatural.getDate()}/${dateNatural.getMonth() + 1}/${dateNatural.getFullYear()}`;
```